README.md

```diff
@@ -46,21 +46,48 @@ Features include:
 Installing
 ==========
 
-There will _soon_ be a release of Swarm on Hackage and in a binary form!
-If you are interested in Haskell or functional programming, you can install
-it from source (see below) or, once it is released, with the Cabal tool.
-For instructions on how to install the binary directly, check the releases.
+**NOTE**: Swarm requires a POSIX-style terminal environment that
+supports `terminfo`.  Linux and MacOS should work fine.  On Windows,
+you will need to use something like the Windows Subsystem for Linux.
+For example, see [the comments on this GitHub
+issue](https://github.com/swarm-game/swarm/issues/53) for instructions
+on building from source on Windows.
```

Comment:
> something like the Windows Subsystem for Linux
I am not aware there are any alternatives.

Once you have WSL2, you have a Linux, so normal installation instructions (including binary) should apply.

Just a nitpick about the wording. :wink: